vagrant_repo/apps/my_app/default/data/ui/nav/default.xml

```diff
@@ -0,0 +1,3 @@
+<nav search_view="search" color="#72231F">
+  <view name="lorem_ipsum" default="true" />
+</nav>
```

Comment:
Fixed with a44768b5e83fabfe91ba8c0859698fab79b9f73b (being the top commit I rewrote history)